README.md

```diff
@@ -25,6 +25,21 @@ Change your `Cargo.toml` to set the crate-type to `cdylib` (this instructs the c
 crate_type = ["cdylib"]
 ```
 
+### Rustup and wasm32-unknown-unknown installation
+
+Our example below will use the `wasm32-unknown-unknown` target. If this is not installed you will need to do so before this example will build. The easiest way to do this is use [`rustup`](https://rustup.rs/).
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Once `rustup` is installed, add the `wasm32-unknown-unknown` target:
+
+```bash
+rustup target add wasm32-unknown-unknown
+```
+
+
```

Comment:
Sorry for the delay here, didn't realize this got left behind... 

> Actually.. should the comment and ### Rustup.. part be up at the top too? Thus this whole bit is at the top? Otherwise just the curl command may look a little odd.

Do you mean to include this code attached here to the `curl` command to be moved up?